src/data/exercises.json

```diff
@@ -0,0 +1,455 @@
+{
+    "total_count": 10,
```

Comment:
I believe this JSON is for a set of exercises but the `<a href="...">` in the content HTML points to a single exercise. Thoughts @Dantemss?